### ISSUE TITLE
No-chop merge debug logging

### ISSUE THREAD

**Issue by cshannon** [apache/accumulo#3767]
As part of the No-chop merge changes in #3761 the file metadata is being modified to store ranges as base 64 format which is hard to read for debugging as it is encoded. We should add a way to make this human readable and printable to the logs for debugging and logging purposes, which will likely involve deserializing the encoded ranges back into a Range object.

**Comment by keith-turner** [comment]
See #3772

**Comment by cshannon** [comment]
Looks like https://github.com/apache/accumulo/pull/3885 might address this or maybe partially

**Comment by keith-turner** [comment]
> Looks like https://github.com/apache/accumulo/pull/3885 might address this or maybe partially

Yeah it is partial, not all file logging is consistent.  I focused on the logging that just logs the file name in #3885 as it was logging incomplete information.  For this issue it may be good to look at making more logging of tablet files use the same methods.

**Comment by ArbaazKhan1** [comment]
I can take a look at this
